portfolio/src/main/webapp/script.js

```diff
@@ -110,3 +109,25 @@ function diamondDescriptionUpdate(hitCount) {
         " nothing more to gain from this."
     }
 }
+
+/**
+* Temporary function that formats and displays comment on page. This does not yet save comments.
+*/
+function displayComment() {
+    fetch('/data').then(response => response.text()).then((dataResponse) => {
```

Comment:
i think i left the other comment in the wrong place, so repeating here:

I think you can use "response => response.json()" and thus avoid the need for the JSON.parse call below?

Reply:
Sorry I missed this! Fixed.